Common/UnitDefinitions/LeakRate.json

```diff
@@ -0,0 +1,52 @@
+{
+    "Name": "LeakRate",
+    "BaseUnit": "PascalQubicMeterPerSecond",
+    "XmlDocSummary": "The leakrate is a measure of the units of volume or mass leaking from a body.",
```

Comment:
I don't quite understand this description. Is there an alternative way to describe it?

From another source: https://www.leybold.com/en/knowledge/vacuum-fundamentals/leak-detection/definition-and-measurement-of-vacuum-leaks

> A leak rate of QL = 1 mbar · l/s is present when in an enclosed, evacuated vessel with a volume of 1 l the pressure rises by 1 mbar per second or, where there is positive pressure in the container, pressure drops by 1 mbar.